src/thematic/4_vegetation/sfp-401_vegopname-terrest_nl/12_appendices.Rmd

```diff
@@ -57,9 +58,9 @@ knitr::include_graphics(path = './media/image11.png')
 
 Grootte van het proefvlak:
 
-Open vegetaties 3 m x 3 m
+Open vegetaties `r params$length_OV_m`m \*`r params$width_OV_m`m
 
-Bossen 16 m x 16 m
+Bossen `r params$length_B_m`m \* `r params$width_B_m` m
 
 2.56 m^2^ = 1% 30 cm x 30 cm = 1%
```

Comment:
```suggestion
```r, results = "asis"
opp_b <- params$length_B_m * params$width_B_m
opp_ov <- params$length_OV_m * params$width_B_m
zijde_b_1perc <- sqrt(0.01 * opp_b)
zijde_ov_1perc <- sqrt(0.01 * opp_ov)
sprintf("Voor bossen komt 1%% overeen met %.2f m x %.2f m.", zijde_b_1perc, zijde_b_1perc)
sprintf("Voor open vegetaties komt 1%% overeen met %.2f m x %.2f m.", zijde_ov_1perc, zijde_ov_1perc)
```

Reply:
= a way to write that sentence while making use of the params fields. Inserting this as an r-chunk with results = "asis" in the knitr chunk options, will write the sentence "Voor bossen komt 1% overeen met 1.60 m x 1.60 m.